pallets/mixer/src/lib.rs

```diff
@@ -320,6 +318,22 @@ pub mod pallet {
 			Ok(().into())
 		}
 
+		// NOTE: Used only for testing purposes
+		#[pallet::weight(0)]
+		pub fn create_new(
+			origin: OriginFor<T>,
+			currency_id: CurrencyIdOf<T>,
+			size: BalanceOf<T>,
+		) -> DispatchResultWithPostInfo {
+			ensure_admin(origin, &Self::admin())?;
+
+			let depth: u8 = <T as merkle::Config>::MaxTreeDepth::get();
+			let mixer_id: T::GroupId = T::Group::create_group(Self::account_id(), true, depth)?;
+			let mixer_info = MixerInfo::<T>::new(T::DepositLength::get(), size, Vec::new(), currency_id);
+			MixerGroups::<T>::insert(mixer_id, mixer_info);
+			Ok(().into())
+		}
+
```

Comment:
Couldn't think of a better way to test new tokens, without creating a new mixer with non-native token

Reply:
This is a good start though, since I think this is the likely next feature to add. We will want to have a path for creating new mixers which requires an upfront deposit paid to some account...

Reply:
This is a good candidate for the next task. Do we want to do a deposit inside `create_new` (or whatever we call it), like inside `deposit`: `T::Currency::transfer(sender, some_account)`?

Reply:
Yea, it would be good to make the up-front deposit parameters governable as well. We can follow the pattern from here: https://github.com/hicommonwealth/edgeware-node/blob/master/modules/edge-treasury-reward/src/lib.rs#L56

Reply:
Though it's possible this is best placed inside the `merkle` pallet as a deposit to create trees.

Reply:
Adding to my first comment --- Maybe we shouldn't allow it to be dispatchable for now, and we can have a trait `ExtendedMixer` that implements the `create_new` function

Reply:
Update: Moved `create_new` to `ExtendedMixer` impl